bus/bus.go

```diff
@@ -303,6 +307,7 @@ type (
 type Bus struct {
 	startTime time.Time
 	masterKey utils.MasterKey
+	network   *consensus.Network
```

Comment:
No need for that. The `ChainManager` can already provide this.

Reply:
Right! Dammit didn't think of that.